internal/processor/parser/app.go

```diff
@@ -2,37 +2,70 @@ package parser
 
 import (
 	"fmt"
+	"net/url"
 	"sync"
 	"time"
 
-	"encoding/json"
 	"github.com/DataDog/datadog-firehose-nozzle/internal/metric"
 	"github.com/DataDog/datadog-firehose-nozzle/internal/util"
 	"github.com/cloudfoundry-community/go-cfclient"
 	"github.com/cloudfoundry/gosteno"
 	"github.com/cloudfoundry/sonde-go/events"
-	bolt "github.com/coreos/bbolt"
 )
 
-var clearCacheDuration = 60
+type appCache struct {
```

Comment:
Do you think we could make this cache more generic. something like this:

```
type Cacher interface {
	Add(key string, value interface{}) error
	Get(key string) (interface{}, bool, error)
	Delete(key string) error
}
```

Reply:
I thought about it, but I didn't think we would reuse that anywhere. I'll isolate it into its own package if you feel strongly about it.